Makefile

```diff
@@ -17,7 +17,7 @@ lint:
 
 .PHONY: mypy
 mypy:
-	mypy --ignore-missing-imports --follow-imports=skip --strict-optional --warn-no-return .
+	mypy --ignore-missing-imports --strict-optional --warn-no-return .
```

Comment:
Could you let us know what this change is for?

Reply:
Ah, yes! If we don't follow the imports, mypy complains when we import User as a parent class for the elasticsearch model class and then call certain properties that it inherits from the parent class.